Orso.Arpa.Domain/UserDomain/Commands/LoginUser.cs

```diff
@@ -66,7 +71,28 @@ public async Task<string> Handle(Command request, CancellationToken cancellation
                     throw new ValidationException(new[] { new ValidationFailure(nameof(request.UsernameOrEmail), "Your email address is not confirmed. Please confirm your email address first") });
                 }
 
-                SignInResult result = await _signInManager.CheckPasswordSignInAsync(user, request.Password, true);
+                SignInResult result;
+
+                if (_identityConfiguration.UseCookies)
+                {
+                    // var passwordOK = await _userManager.CheckPasswordAsync(user, request.Password);
+                    // if (passwordOK)
+                    // {
+                    //     await _signInManager.SignInWithClaimsAsync(user, false, new List<Claim>
```

Comment:
in order to add claims, this function can be used but not preferred since it does not return action result which we rely on later